players/templates/players/base.html

```diff
@@ -20,20 +20,40 @@
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css">
 
         {% block styles %}
+        <style>
+          @import url('https://fonts.googleapis.com/css?family=Baloo|Source+Sans+Pro');
+          html { font-family: 'Source Sans Pro', sans-serif; }
+          h1, h2, h3 { font-family: 'Baloo', cursive; }
+          body { font-size: 1.5em; }
+          p { line-height: 1.3em; }
+
+          @media screen and (min-width: 24.15em) { /* 21em "gate" * 1.15 font-size */
+            p { line-height: calc(1.3em + (1.5 - 1.3) * ((100vw - 21em) / (35 - 21))); }
+          }
+
+          @media (min-width: 40.25em) { /* 35em "gate" * 1.15 font-size */
+            p { line-height: 1.5em; }
+          }
+        </style>
         {% endblock %}
     </head>
 
     <body>
-        <nav class="navbar navbar-default">
+        <nav class="navbar navbar-default" style="background-color: white; background-image: none">
```

Comment:
Since this is a preview version, I don't really care about code quality as we're changing it anyway so I'm fine with this, but down to someone else I guess: @j4mesholland @CelineBoudier ? After the conference this will get dropped.

Reply:
It will be replaced in the next few months by a React UI